MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -1373,4 +1404,9 @@ the Select` button. When clicked it closes all the previously selected tabs -->
     <!-- This string is displayed as a button below the slow script dialog description message. When clicked
        it continues to wait for execution of the web page script that caused the slow down. -->
     <string name="slow_script_dialog_action_wait">Wait</string>
+
+
+    <!-- This string is displayed in the title of the quick WebXR blocking dialog, accesed from the VR icon in the URL bar.
+     '%1$s' will be replaced at runtime with the app's name. -->
+    <string name="webxr_block_dialog_message">\'%1$s\' wants to access WebXR api</string>
```

Comment:
@MortimerGoro 
Could we avoid straight quotes and the escaping here? Also, why is *api* lowercase? Normally it's uppercase, and it's API in another string in the same commit.

```
‘%1$s’ wants to access WebXR API
```

Or

```
“%1$s” to access WebXR API
```

Reply:
yes, we can avoid the quotes and change API to uppercase. Do you want me to fix that in FxR master or can we fix it here?

Reply:
You need to fix it in FxR, the fix will propagate here once it lands.